### PR TITLE
Test quiz section styled and alert added

### PR DESCRIPTION
**Pull request by Simran2337**
image

The spacing and alert for not selecting any option and clicking to next question button has been added.

### CODE REVIEW

File: quiz.html [hunk 1]
```diff
@@ -1,34 +1,42 @@
...
         <img src="yoga.png" alt="yoga-logo" class="yoga-logo">
-       <div id="questions"></div>
-       <div>
-        <button class="nextQues" class="ml-5" onclick = "nextQues()">Next Question <span>&#10132;</span></button>
-       </div>
+        <br>
+        <div id="questions">
+        </div>
+        <br>
+        <br>
```
Comment (tarunsinghofficial) [inline quiz.html:30]: Instead of using <br>, use bootstrap spacing.

File: quiz.js [hunk 2]
```diff
@@ -64,81 +64,80 @@ let questions = [
...
-function show(i){
+let points = 0;
+function show(i) {
 
 
-  document.getElementById("questions").innerHTML=`
-  <h2 class="pl-5">Q${i+1}.${questions[i].question}</h2>
+  document.getElementById("questions").innerHTML = `
+  <h2 class="pl-5">Q${i + 1}.${questions[i].question}</h2>
+  <br><br>
```
Comment (tarunsinghofficial) [inline quiz.js:86]: Instead of using <br>
, use bootstrap spacing.

**Review by tarunsinghofficial — CHANGES_REQUESTED** [review]

**Comment by Simran2337** [comment]
@tarunsinghofficial  okay sure

**Comment by Simran2337** [comment]
@tarunsinghofficial  Changes done!!

**Pull request merged**